content/docs/rfcs/13/README.md

```diff
@@ -85,7 +85,8 @@ To perform pagination, each `WakuMessage` stored at a node running the `WakuStor
 
 RPC call to query historical messages.
 
-- The `topics` field MUST indicate the list of topics to query.
+- The `topics` field MUST indicate the list of topics to query. 
```

Comment:
I wonder why we call it `topics` field instead of `contentTopics`? I can't really why this is the case. Would it make sense to explicitly call it contentTopics instead?

Reply:
I agree, with `contentTopics`  the relation with waku message `contentTopic` would be more clear.

Reply:
> I wonder why we call it `topics` field instead of `contentTopics`? I can't really see why this is the case. Would it make sense to explicitly call it contentTopics instead?

Done, btw, to my understanding there is no one to one mapping bw the protobuf field names and the actual Nim implementation as long as the types match. So, shall we also rename this field in the Nim implementation of history queries  or it is unnecessary? Let me know and I will open an issue for it.